lerna.json

```diff
@@ -4,7 +4,7 @@
   "version": "independent",
   "command": {
       "publish": {
-        "allowBranch": "development"
+        "allowBranch": "master"
```

Comment:
The reasoning behind allowing development was the inability to push to master, has this been changed?

Reply:
no, I think this was a mistake